tft_bot/helpers/system_helpers.py

```diff
@@ -240,3 +240,37 @@ def determine_tesseract_ocr_install_location() -> str:
     logger.debug(f"Tesseract-OCR location determined to be: {tesseract_ocr_path}")
 
     return tesseract_ocr_path
+
+
+def determine_deceive_install_location() -> str | None:
+    """
+    Tries to determine the install location of Deceive.
+
+    Returns:
+        If successful, a string of the determined location, if not, None.
+    """
+    # Check downloads folder first. Might just find it
+    downloads_path = str(pathlib.Path.home() / "Downloads")
+    for root, dirs, files in os.walk(downloads_path):   # pylint: disable=unused-variable
+        for file in files:
+            if "Deceive.exe" in file:
+                logger.debug("Found Deceive in Downloads folder")
+                return os.path.join(root, file)
+
+    # Search registry now
+    key_to_read = r"SOFTWARE\Microsoft\Windows NT\CurrentVersion\AppCompatFlags\Compatibility Assistant\Store"
+    k = winreg.OpenKey(winreg.HKEY_CURRENT_USER, key_to_read, 0, winreg.KEY_READ)
```

Comment:
Since this is checking the "recent apps" registry entries, I'd argue it might be worth commenting this since it isn't guaranteed to be in here if the user hasn't run it yet and/or it's not there

Reply:
By commenting, do you mean commenting it out (i.e. removing) or just documenting it?

Reply:
Just adding a comment! 😄